Report/Chapters/Requirements.tex

```diff
@@ -1,26 +1,76 @@
 \chapter{Requirements Analysis}
 
-The first step in creating an application or program is defining the requirements. In our discussions with the client, he emphasized that the main goal of this project was making a user-friendly interface.
+The first step in creating an application or program was defining the requirements. In order to do this, we planned meetings with our client. Against our expectations the client did not have many demands to the project. He wanted us to be free in the design and implementation. The main goal of the project was not creating a program, which is 100\% finished and could be used right away, but to explore the possibilities of the client's ideas (a proof of concept).
```

Comment:
I think this is putting it neutrally and without causing offence. In other words, I like it.